From 0dd53533e20d2948351a99ec5336fbc9b82b226a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Amadeusz=20S=C5=82awi=C5=84ski?= <amade@asmblr.net>
Date: Wed, 5 Feb 2020 21:05:28 +0100
Subject: [PATCH] Increase permitted length of OSC
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

hyperlink feature used by some terminals requires lots of characters
https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda#length-limits
mentions around 2083 characters, set it to a bit more.

Bug: 57718

Signed-off-by: Amadeusz Sławiński <amade@asmblr.net>
---
 window.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/window.h b/window.h
index 11d2a9e..2d7e0f5 100644
--- a/window.h
+++ b/window.h
@@ -237,7 +237,7 @@ struct win
   char	 w_vbwait;
   char	 w_norefresh;		/* dont redisplay when switching to that win */
 #ifdef RXVT_OSC
-  char	 w_xtermosc[5][MAXSTR];	/* special xterm/rxvt escapes */
+  char	 w_xtermosc[5][2560];	/* special xterm/rxvt escapes */
 #endif
   int    w_mouse;		/* mouse mode 0,9,1000 */
   int    w_extmouse;		/* extended mouse mode 0,1006 */
-- 
2.23.0

